### PR TITLE
Add ponyo pip install 

### PR DESCRIPTION
**Pull request by ajlee21**
This PR does the following:

1. Adds ponyo to pip install in `environment.yml` file
2. Adds additional pytests for directory setup and reproducible vae training
3. Updates readme to clarify how to run the analysis in this repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,38 +32,45 @@ There are 2 analyses using Pseudomonas dataset in the `Pseudomonas` directory an
...
+# conda version 4.6.12
+conda env create -f environment.yml
+
+conda activate simulate_expression_compendia
+
+pip install -e .
+```
 3. Create a new analysis folder in the main directory. This is equivalent to the `Pseudomonas` directory
-4. Copy `Pseudomonas_sample_lvl_sim.ipynb` and `Pseudomonas_experiment_lvl_sim.ipynb` into your analysis folder. 
+4. Copy `Pseudomonas_sample_lvl_sim.ipynb` or `Pseudomonas_experiment_lvl_sim.ipynb` into your analysis folder depending on if you would like to use the sample level or experiment level simulation approach.
```
Comment (ben-heil) [inline README.md:69]: You may want to add a sentence or link to what the sample level and experiment level simulation approaches are

File: README.md [hunk 2]
```diff
@@ -32,38 +32,45 @@ There are 2 analyses using Pseudomonas dataset in the `Pseudomonas` directory an
...
-## How to run this simulation using your own data
+## How to run using your own data
 
 In order to run this simulation on your own gene expression data the following steps should be performed:
 
-First you need to set up your local repository: 
+First you need to set up your local repository and environment: 
 1. Clone the `simulate-expression-compendia` repository
-2. Set up conda environment using the command above
+2. Set up conda environment by running the following command in the terminal:
```
Comment (ben-heil) [inline README.md:59]: It may help to link to the miniconda install link here for people who don't know what `conda` is

Comment (ajlee21) [reply README.md:59]: Good point. I will add this. Definitely modeled these new instructions after @miltondp 's nice example in phenoplier!

File: README.md (hunk 1 above)
Comment (ajlee21) [reply README.md]: Thats a great idea. I have added a linkout to ponyo which has text in the functions to explain the method in more detail

**Pull request merged**